OpenSim/Auxiliary/auxiliaryTestFunctions.h

```diff
@@ -333,26 +332,6 @@ inline bool revertToVersionNumber1(const std::string& filenameOld,
     return changedVersion;
 }
 
-// Add number of rows (nRows) and number of columns (nColumns) to the header of
-// the STO file. Note that nColumns will include time, so it will be number of
-// columns in the matrix plus 1 (for time).
-inline void addNumRowsNumColumns(const std::string& filenameOld,
```

Comment:
I think this was used to create `.sto` files in the earlier format so the the `Storage` class could read it. I just want to double check that his is no longer necessary because `Storage` now uses the `STOFileAdapter` internally, correct?